### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by AnicetFantomas**
In this pull request we performed :

- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,42 @@ SELECT (name, escapes_attempts) FROM animals WHERE weight_kg >10.5;
...
 SELECT * FROM animals WHERE weight_kg >=10.4 AND weight_kg <=17.3;
+
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species='pokemon' WHERE species='unspecified';
```
Comment (KDlamini) [inline queries.sql]: The default value for species if not specified is NULL in the database. Therefore, your query `WHERE species='unspecified'` is invalid.

- You can optimize your query by using `WHERE species IS NULL`.

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,42 @@ SELECT (name, escapes_attempts) FROM animals WHERE weight_kg >10.5;
...
+-- How many animals have never tried to escape?
+SELECT COUNT(name) FROM animals WHERE escapes_attempts = 0;
+-- What is the average weight of animals?
+SELECT AVG(weight_kg) from animals;
+-- Who escapes the most, neutered or not neutered animals?
+SELECT MAX(escapes_attempts) from animals;
+-- What is the minimum and maximum weight of each type of animal?
+SELECT species, (MIN(weight_kg), MAX(weight_kg)) FROM animals GROUP BY species;
+-- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
+SELECT species, AVG(escapes_attempts) from animals where EXTRACT(YEAR FROM date_of_birth) between 1990;
```
Comment (KDlamini) [inline queries.sql]: The date must be between 1990 and 2000, kindly append `AND 2000` at the end of your query to get the desired date range. The  `BETWEEN` query must have a minimum and maximum value to set a range.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @AnicetFantomas 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Correct PR title
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @AnicetFantomas, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**